{"name": "Логин с неверным паролем", "status": "passed", "start": 1729183622391, "stop": 1729183624458, "uuid": "ddd9c967-c7b0-45d2-b10c-78cc6028d761", "historyId": "1d2686c181cb4de7e6d07d3e9826b8f9", "testCaseId": "1d2686c181cb4de7e6d07d3e9826b8f9", "fullName": "tests.test_login.TestLogin#test_login_with_incorrect_pass", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-2V6LHVH"}, {"name": "thread", "value": "5276-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}